haystack/database/elasticsearch.py

```diff
@@ -236,6 +236,62 @@ def get_all_labels(self, index: Optional[str] = None, filters: Optional[Dict[str
         labels = [Label.from_dict(hit["_source"]) for hit in result]
         return labels
 
+    def get_all_labels_aggregated(self, index: Optional[str] = None, filters: Optional[Dict[str, List[str]]] = None) -> List[Aggregated_Label]:
```

Comment:
I would suggest moving this function out of the ElasticsearchDocumentStore. Otherwise, we need to re-implement it for every other DocumentStore. IMO it would be better if it would take List[Labels] as input and returns List[Aggregated_Label].